### PR TITLE
feat: add direct url and local path, rework downloadAgent flow

### PR DESCRIPTION
**Pull request by bodyangug**

### CODE REVIEW

**Review by RomanDavlyatshin — APPROVED** [review]

File: agent-runner-common/src/main/kotlin/com/epam/drill/agent/runner/AgentLoader.kt [hunk 1]
```diff
@@ -1,18 +1,3 @@
-/**
```
Comment (RomanDavlyatshin) [inline agent-runner-common/src/main/kotlin/com/epam/drill/agent/runner/AgentLoader.kt:1]: gh action is failing for both files on license-check stage

**Review by RomanDavlyatshin — CHANGES_REQUESTED** [review]
GH acti

File: agent-runner-common/src/main/kotlin/com/epam/drill/agent/runner/AgentLoader.kt (hunk 1 above)
Comment (bodyangug) [reply agent-runner-common/src/main/kotlin/com/epam/drill/agent/runner/AgentLoader.kt]: done

**Comment by RomanDavlyatshin** [comment]
We'll remove this component from lib-jvm-shared repo
See same PR at https://github.com/Drill4J/agent-runner/pull/12

**Pull request closed without merging**